components/mixins/experience-event/events/visit-webpage.schema.json

```diff
@@ -45,12 +45,12 @@
                 "xdm:queryParamaters": {
                   "title": "Query Paramaters",
                   "type": "string",
-                  "description": "Query Paramters used for the web page."
+                  "description": "Query Paramters used for the web page. This is normally the value of a string after ? in the url."
```

Comment:
Query Paramters used for the web?page. This is normally the value of a string after `?` in the url. => Query Parameters used for the webpage. This is normally the value of a string after `?` in the url.